tools/mo/unit_tests/mo/ops/activation_test.py

```diff
@@ -144,3 +144,33 @@ def test_activation_swish_infer(self):
             self.assertEqual(res_shape[i], value)
         for i, value in enumerate(exp_value):
             self.assertAlmostEqual(res_value[i], value)
+
+    def test_activation_softsign_infer(self):
+        graph = build_graph(self.nodes_attributes,
+                            edges=[
+                                ('node_1', 'activation_node'),
+                                ('activation_node', 'node_3')
+                            ],
+                            update_attributes={
+                                'node_1': {
+                                    'value': np.array([1.0, -1.0, 3.5, -5.8])
+                                },
+                                'activation_node': {
+                                    'op': 'SoftSign',
+                                    'operation': SoftSign.operation
+                                },
+                                'node_3': {
+                                    'value': None
+                                }
+                            })
+        graph.graph['layout'] = 'NCHW'
+        activation_node = Node(graph, 'activation_node')
+        SoftSign.infer(activation_node)
+        exp_shape = np.array([4])
+        res_shape = graph.nodes['node_3']['shape']
+        res_value = graph.nodes['node_3']['value']
+        exp_value = np.array([0.5, -0.5, 0.7777777777777, -0.85294117647])
+        for i, value in enumerate(exp_shape):
+            self.assertEqual(res_shape[i], value)
+        for i, value in enumerate(exp_value):
+            self.assertAlmostEqual(res_value[i], value)
```

Comment:
I would suggest to use some helpers like `np.array_equal`/`np.allclose` instead of assert in loop.